web/src/components/LoginModal/LoginModal.js

```diff
@@ -0,0 +1,189 @@
+import { useState } from 'react'
+import Dialog from '@material-ui/core/Dialog'
+import Tab from '@material-ui/core/Tab'
+import Tabs from '@material-ui/core/Tabs'
+import InputTextForm from 'src/components/InputTextForm'
+import OutBound from 'src/components/OutBound'
+import { Form, Submit } from '@redwoodjs/forms'
+import { useAuth } from '@redwoodjs/auth'
+import { useFlash } from '@redwoodjs/web'
+import { Link, routes } from '@redwoodjs/router'
+import { subscribe } from 'src/helpers/subscribe'
+
+const LoginModal = ({ open, onClose, shouldStartWithSignup = false }) => {
```

Comment:
The login modal
![image](https://user-images.githubusercontent.com/29681384/101236984-5853b600-3729-11eb-97c2-6ee7a4f73e2e.png)